cpp/src/backends/process/model_worker.cc

```diff
@@ -1,6 +1,13 @@
 #include "src/backends/process/model_worker.hh"
 
+namespace fs = std::filesystem;
```

Comment:
why do we need this line when we are defining aliases below?

Reply:
thanks, copy-paste error